internal/cli/iam/organizations/apikeys/accesslists/access_lists.go

```diff
@@ -36,3 +43,68 @@ func Builder() *cobra.Command {
 
 	return cmd
 }
+
+// shouldUseWhitelist returns true when it is service Ops Manager and all versions below 5.0
+// returns false when Atlas, Cloud Manager or Ops Manager 5+.
+func shouldUseWhitelist(s store.ServiceVersionGetter) (bool, error) {
+	if config.Service() != config.OpsManagerService {
+		return false, nil
+	}
+
+	version, err := s.GetServiceVersion()
+	if err != nil {
+		return false, err
+	}
+	versionParts := strings.Split(version.Version, ".")
+
+	const maxVersionParts = 3
+
+	if len(versionParts) > maxVersionParts {
+		versionParts = versionParts[0:3] // ops manager versions are not semantic this is converting it to x.y.z
+	}
```

Comment:
[np]: If we were doing testing with an rc, I think this would break (new version would keep the `-4cX` - a trim end to remove rc might be good here.

I'm really not sure how likely this scenario is however.

Reply:
just tested and `semver.NewVersion("5.0.0-rc0")` parses correcly, no errors

Reply:
@gssbzn Will remember this better, last release it was `5.0` as we made sure it was in the first rc, however after we decided we don't need to block the first rc on the version manifest.

If the semver is working and getting the magor version, I think we're good, as it will be correct for the final rc

Reply:
I'm not sure I follow @colm-quinn's question here, version manifest is for server versions, this is for ops manager version, we need to drop anything after the last dot(.) so @fmenezes is right here that this should work

Reply:
I might be misreading, the original split is: 
`versionParts := strings.Split(version.Version, ".")` ,giving `[5,0,0-rc0]` 

In that case, which means that the `-rc0` text isn't dropped by: `newVersion := strings.Join(versionParts, ".")`

Reply:
yes, and that's valid, the problem is `5,0,0-rc0.something` .something is not valid and what we want to drop

Reply:
Ah got it, thanks!